OpenRA.Mods.Common/Traits/ReloadAmmoPool.cs

```diff
@@ -0,0 +1,58 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Reloads an ammo pool.")]
+	public class ReloadAmmoPoolInfo : ConditionalTraitInfo
+	{
+		[Desc("Reload ammo pool with this name.")]
+		public readonly string AmmoPool = "primary";
```

Comment:
Can we get a RulesetLoaded check to make sure this points at a valid ammo pool?  This will make sure that we catch errors in the linter or at least during game load, instead of crashing when the unit is built.

Reply:
Added.